### PR TITLE
run test playbook from CI

### PR DESCRIPTION
**Pull request by afeld**
The one-time setup steps, for reference:

1. [Disabled public builds](https://circleci.com/docs/2.0/oss/#overview) in CircleCI
1. Created the SSH key for the `deployer` user

    ```shell
    ssh-keygen -t rsa -b 4096 -C circleci@planninglabs.nyc -f deployer
    ```

1. [Added the SSH key to CircleCI](https://circleci.com/docs/2.0/add-ssh-key/)
1. [Created a new (read-only) DigitalOcean API token](https://www.digitalocean.com/docs/api/create-personal-access-token/)
1. [Set the `DO_API_TOKEN` environment variable in CircleCI](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project)

### CODE REVIEW

**Comment by afeld** [comment]
Build is [failing](https://circleci.com/gh/NYCPlanning/labs-infrastructure/15) as expected. Should pass once the base playbook (from this branch) is run across all Droplets.

**Comment by afeld** [comment]
* [x] Consider [disabling public builds](https://circleci.com/docs/2.0/oss/#overview)

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,6 +19,18 @@ jobs:
...
+          name: Run the test playbook against all Droplets
+          # redact IP addresses
+          # https://gist.github.com/afeld/e53ac65e122ccd27547f2d2a54755f8b
+          command: |
+            pipenv run ansible-playbook -i digital_ocean.py -u deployer playbooks/test.yml | \
+            sed -E 's/[0-9]+\.[0-9]+\.[0-9]+\.([0-9]+)/███.███.███.\1/g'
+      - run:
+          name: Run the base playbook in check mode against all Droplets
+          command: |
+            ansible-playbook -i digital_ocean.py -l ubuntu-test -u deployer -C playbooks/base.yml | \
```
Comment (afeld) [inline .circleci/config.yml]: Whoops, remove `-l`.

**Comment by afeld** [comment]
Made the builds private, as a precaution.

**Review by allthesignals — APPROVED** [review]

**Pull request merged**